docs/configuration.md

```diff
@@ -1704,6 +1704,82 @@ implicit protected val td: TildeArrow {
   type Out = RouteTestResult } = TildeArrow.injectIntoRoute
 ```
 
+### `newlines.sometimesBeforeColonInMethodReturnType`
```

Comment:
```suggestion
### `newlines.colonInMethodReturnTypeOnOverflow`
```
?

`Sometimes` has weird undermined sound to it

Reply:
Or is it not a new config?

Reply:
this is an old parameter but was never mentioned in the docs. since it's used in the new logic, added it.